library/spdm_crypt_lib/fips/libspdm_selftest_rsa_ssa.c

```diff
@@ -0,0 +1,208 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "internal/libspdm_crypt_lib.h"
+
+#if LIBSPDM_FIPS_MODE
+
+/**
+ * RSA_SSA(RSASSA-PKCS1 v1.5) self_test
+ **/
+bool libspdm_fips_selftest_rsa_ssa(void)
+{
+    bool result;
+    result = true;
```

Comment:
For C89 compatibility we don't allow procedural statements until all variables have been declared for a scope. So I would make this `bool result = true;`.

Reply:
Thanks. I will remember it.